### ISSUE TITLE
Technical Issue #1

### ISSUE THREAD

**Issue by KentPendergast** [dmcnamara1/Project-Repository#4]
<img width="1235" alt="Screen Shot 2022-11-12 at 4 28 08 PM" src="https://user-images.githubusercontent.com/112450902/201495245-21786f07-3503-4974-a245-733bff8470b7.png">

On your More Synpact page, the video preview box was small and didn't fit your page that well.  You could reduce the outline shapes you have around it and increase the size of your preview box a bit to give a better experience.

**Comment by dmcnamara1** [comment]
Yes I agree, I tried to but couldn't figure it out so I'll definitely go into the next submission with this being tweaked. Thanks!
